### DISCUSSION TITLE
I removed the actions plugin from the GitHub marketplace and when I re-release it, it prompts Name must be unique. Cannot match an existing action, user or organization name.

### DISCUSSION THREAD

**Discussion by VAllens** [community/community#88544] [Actions]
### marketplace, actions

Bug

### Body

I removed the actions plugin from the GitHub marketplace and when I re-release it, it prompts `Name must be unique. Cannot match an existing action, user or organization name`.
![image](https://github.com/community/community/assets/5094922/79203146-913f-421c-9212-344252aa52a1)

**Comment by anandnd** [comment]
I am having same issue, @VAllens pls let me know if you found a way out, thanks!
